recipe_scrapers/ethanchlebowski.py

```diff
@@ -0,0 +1,41 @@
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, get_yields, normalize_string
+
+
+class EthanChlebowski(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "ethanchlebowski.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def instructions(self):
+        return self.schema.instructions().split("\n")
```

Comment:
This might seem a bit odd, but currently we return a single string from the `instructions` method, so to be consistent with the other scrapers, we should remove the `.split("\n")` here.

We have a plan in #283 to add a method `instructions_list` that returns items in list format.

@hhursev @bfcarpio I think I'll go ahead and add the `instructions_list` method soon and migrate all the scrapers to that (with a default implementation of `instructions` that newline-joins the results).

Reply:
I was suspecting something like that was going on, fixed

Reply:
Thanks!